### PR TITLE
Introduction | Kajori4

### PR DESCRIPTION
**Pull request by Kajori4**
I followed the rules mentioned.Please dont mark invalid, did all the rules.This pr would help a lot.:)

### CODE REVIEW

**Review by NishkarshRaj — CHANGES_REQUESTED** [review]
Hey @Kajori4 

Need some changes before merging!

1. Introduction - Your website link has my portfolio linked. Change that or remove website if you don't have your own portfolio

2. Blog - You are meant to create a blog on technology or programming and not on yourself. Please stick to the rules

**Comment by NishkarshRaj** [comment]
> I followed the rules mentioned.Please dont mark invalid, did all the rules.This pr would help a lot.:)

No! We won't mark your PR invalid. We would love to help you make a valid PR 😄

**Comment by Kajori4** [comment]
I updated. Hope this helps! I tried all

**Review by Kajori4 — COMMENTED** [review]
I updated required changed. Hope this helps

File: blogs/Kajori_blog.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+* X.head() 
+
+
+
+
+
+## Table of Content
+
+| Topic Name | Contributor |
```
Comment (NishkarshRaj) [inline blogs/Kajori_blog.md]: This lines must be present in README.md file not in this

File: blogs/Kajori_blog.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+* import pandas as pd
+* import numpy as np
+* np.set_printoptions(precision=4)
+* from matplotlib import pyplot as plt
+* import seaborn as sns
+* sns.set()
+* from sklearn.preprocessing import LabelEncoder
+* from sklearn.tree import DecisionTreeClassifier
+* from sklearn.model_selection import train_test_split
+* from sklearn.metrics import confusion_matrix
```
Comment (NishkarshRaj) [inline blogs/Kajori_blog.md]: Code must always be enclosed within triple ticks.

If you are new to markdown, follow these guidelines: https://github.com/NishkarshRaj/Markdown-for-Documentation/tree/master/Code

**Review by NishkarshRaj — CHANGES_REQUESTED** [review]
I have added few comments. Make necessary changes and get back. If you need any help, we are here to resolve all doubts.

**Comment by Kajori4** [comment]
It's now done i guess sir

**Comment by NishkarshRaj** [comment]
@Kajori4 congratulations! 💯 
Amazing work on the changes! 👍  

Merged with master 😄

**Pull request merged**